eval/src/vespa/eval/tensor/serialization/dense_binary_format.cpp

```diff
@@ -50,16 +51,16 @@ size_t decodeDimensions(nbostream & stream, std::vector<Dimension> & dimensions)
 }
 
 template<typename T, typename V>
-void decodeCells(nbostream &stream, size_t cellsSize, V &cells) {
+void decodeCells(nbostream &stream, size_t cellsSize, ArrayRef<V> cells) {
```

Comment:
consider going back to using vector to reduce changes

Reply:
done